public/API/API_GetGame.php

```diff
@@ -22,29 +22,35 @@
  *  string     Released                   release date information for the game
  */
 
+use App\Platform\Models\Game;
+
 $gameID = (int) request()->query('i');
 
-$game = getGameData($gameID);
+$game = Game::with('system')->find($gameID);
+
+if ($game) {
 
-if ($game === null) {
+    $gameData =
+         [
+            'Title' => $game->Title,
+            'GameTitle' => $game->Title,
+            'ConsoleID' => $game->ConsoleID,
+            'ConsoleName' => (string) $game->system->Name,
+            'Console' => (string) $game->system->Name,
+            'ForumTopicID' => $game->ForumTopicID,
+            'Flags' => 0, // Always '0'
+            'GameIcon' => $game->ImageIcon,
+            'ImageIcon' => $game->ImageIcon,
+            'ImageTitle' => $game->ImageTitle,
+            'ImageIngame' => $game->ImageIngame,
+            'ImageBoxArt' => $game->ImageBoxArt,
+            'Publisher' => $game->Publisher,
+            'Developer' => $game->Developer,
+            'Genre' => $game->Genre,
+            'Released' => $game->Released,
+        ];
+
+    return response()->json($gameData);
+} else {
```

Comment:
`else` can be omitted by inverting the condition (`if (!$game)`) and make it an early return.

Reply:
Good catch, updated